packages/markerclustererplus/src/cluster-icon.ts

```diff
@@ -396,17 +431,25 @@ export class ClusterIcon extends OverlayViewSafe {
    * @return The CSS style text.
    */
   private createCss_(pos: google.maps.Point): string {
-    return [
-      `z-index: ${this.cluster_.getMarkerClusterer().getZIndex()}`,
-      "cursor: pointer",
-      `position: absolute; top: ${pos.y}px; left: ${pos.x}px`,
-      `width: ${this.width_}px; height: ${this.height_}px`,
-      "-webkit-user-select: none",
-      "-khtml-user-select: none",
-      "-moz-user-select: none",
-      "-o-user-select: none",
-      "user-select: none"
-    ].join(";");
+    return (
+      toCssText({
```

Comment:
might be cleaner to the defaults in the object converted to text?

Reply:
Didn't get what you mean

Reply:
```
toCssText({
  "z-index": `${this.cluster_.getMarkerClusterer().getZIndex()}`,
  top: coercePixels(pos.y),
  left: coercePixels(pos.x),
  width: coercePixels(this.style.width),
  height: coercePixels(this.style.height),
  cursor: "pointer",
  position: "absolute",
  "-webkit-user-select": "none",
  "-khtml-user-select": "none",
  "-moz-user-select": "none",
  "-o-user-select": "none",
  "user-select": "none"
})
```

kind of a nit

Reply:
I used concatenation here just because it's faster, and `toCssText` is helpful only when some properties might not be defined. Here all fields are static I don't see any sense to use that method. But this up to maintainer, if you think here consistency and readability are more important than my pros so I will do as you say.

Reply:
maybe just hardcode as single string then without join.

Reply:
used `toCssText` for rest of statements. Look to the latest commit.